apps/wildfires/src/app/pages/about/index.tsx

```diff
@@ -70,14 +70,24 @@ export default function About() {
               >
                 Benefits Navigator
               </a>
-              , a suite of online tools dedicated to helping individuals and
-              families maximize their public benefits and chart permanent
-              pathways out of poverty.
+              , a suite of online tools that helps case managers and their clients to quickly
+              navigate and access the complex arena of federal, state and local public benefits
+              and refundable tax credits. The Benefit Navigator has helped people access over
+              $125 million in additional benefits and feel confident to increase their earned
+              income by an average of 25%.
             </li>
             <li className="list-disc">
-              <span className="font-bold">Mayor Karen Bass's Office</span>:
-              Leads the nation's second largest city with urgency to deliver
-              results for all Angelenos.
+              <span className="font-bold">The Mayor’s Fund for Los Angeles</span>:
+              In support of Mayor’s Karen Bass' work to reduce and end homelessness
+              in Los Angeles, the Mayor’s Fund for Los Angeles is focused on preventing
+              Angelenos from becoming homeless. It has brought together business, philanthropy,
+              nonprofits, and local government to help keep Angelenos housed. Through outreach,
+              case management and expanded legal services the Mayor’s Fund's We Are LA program
```

Comment:
regular, uncredentialed comma
```suggestion
              case management and expanded legal services, the Mayor’s Fund's We Are LA program
```